contracts/id.tmy/src/id.tmy.cpp

```diff
@@ -115,18 +115,22 @@ namespace idtmy
       {
          throwError("TCON1000", "This people username is already taken");
       }
-      
-      //Set the resource limits for the new account
-      eosiobios::bios::resource_config_table _resource_config("eosio.bios"_n, "eosio.bios"_n.value);
-      auto config = _resource_config.get_or_create(get_self(), eosiobios::bios::resource_config());
-      config.total_cpu_weight_allocated = this->initial_cpu_weight_allocation;
-      config.total_net_weight_allocated = this->initial_net_weight_allocation;
-      _resource_config.set(config, get_self());
+      // If the account name exists, set resources and return
+      auto existing_person = _people.find(random_name.value);
```

Comment:
this check is not needed. this is because `newaccountaction.send()` will fail if the account name already exists.